toonz/sources/stdfx/igs_line_blur.cpp

```diff
@@ -3049,7 +3049,7 @@ void pixel_line_root::exec09_same_way_expand(
     }
   }
 
-  /* 各端点(one,anohter)ごと
+  /* 各端点(one,ananother)ごと
```

Comment:
This appears to be a typo fix that still leaves the word typo'd but in another form.
As a comment not a high priority but should just be 'another'.

Reply:
Apologies. Fixed in 69208c7